credsweeper/file_handler/file_path_extractor.py

```diff
@@ -105,3 +106,12 @@ def check_exclude_file(cls, config: Config, path: str) -> bool:
         if Util.get_extension(path) in config.exclude_extensions:
             return True
         return False
+
+    @classmethod
+    def check_file_size(cls, config: Config, path: str) -> bool:
+        if config.size_limit is None:
+            return False
+        if os.path.getsize(path) > config.size_limit * 1024 * 1024:  # Convert size_limit to byte
+            return True
+        else:
+            return False
```

Comment:
The line is not covered with a unit test.

Reply:
I covered them with f35951f.

Thank you for review.